src/scripts/UI/arena_menu.gd

```diff
@@ -0,0 +1,67 @@
+extends Control
+
+const nef_head_preload = preload("res://scenes/NPC/Enemies/nef_head_enemy.tscn")
+
+var has_open : bool
+var has_pause : bool
+var check_is_open : bool = false
```

Comment:
Move these 3 basic variables to after line 15. Remember the one code row between each different variable type, e.g.:

```py
var is_false : bool = false
var is_true : bool = true

var real_number : float = 0

var name : string = "John Doe"
var city : string = "New York"
```

Reply:
In other words, ALL variables should be defined after _export var_'s, and _export var_'s are defined after _onready var_'s. In the following way:

```py
const EXAMPLE_CONST = null

onready var example_onready_var = null

export var example_export_var = null

var example_var = null
```

Reply:
In other words, these variable definitions from this:
```py
var has_open : bool
var has_pause : bool
var check_is_open : bool = false

onready var arena_button = $buttons/fight1
onready var player = get_parent().get_parent()
onready var exit_button = $buttons/ExitButton

var bad_guy_instance = null
var nodes = null
```
To this:
```py
onready var arena_button = $buttons/fight1
onready var player = get_parent().get_parent()
onready var exit_button = $buttons/ExitButton

var bad_guy_instance = null
var nodes = null

var has_open : bool
var has_pause : bool
var check_is_open : bool = false
```